docs: added documentation (#800)
bcgov/CONN-CCBC-portal: docs/process_to_update_banners.md

### Process to update banner message 

> To be able to communicate with customers and convey messages, special area of the initial and dashboard pages was assigned (over the page title). Message in that area can be enabled, disabled or updated via Growthbook web application. GrowthBook is an open source platform that gives any company the power of a customized feature flagging.

Reviewer: ```suggestion
To be able to communicate with customers and convey messages, a special area of the initial and dashboard pages was assigned (over the page title). The message in that area can be enabled, disabled or updated via the Growthbook web application. GrowthBook is an open source platform that gives any company the power of a customized feature flagging.
```

Two feature flags are supported:
- `open_intake_alert`
- `closed_intake_alert`

> Both feauture flags defined as JSON objects with the following structure:

Reviewer: ```suggestion
Both feature flags are defined as JSON objects with the following structure:
```

{"variant": "success", "text": "Applications are now being accepted until [DATE].", "displayOpenDate" : true, "dateTimestamp": "2022-10-01"}

"variant" can be one of the following:
- "success"
- "info"
- "warning"
- "danger"

> "text" is text of the message; it can have merge code `[DATE]` which will be replaced with the value provided in "dateTimestamp" attribute if "displayOpenDate" attribute is set to `true`

Reviewer: ```suggestion
"text" is the text of the message; it can have the merge code `[DATE]` which will be replaced with the value provided in the "dateTimestamp" attribute if the "displayOpenDate" attribute is set to `true`
```

Reviewer: - Not familiar with the "merge code" terminology. Is there another word for that? "placeholder variable" maybe?
- I thought the date was retrieved from the `next_intake` function in the database, is that not the case?

Author: - merge code is standard term (i.e. https://support.microsoft.com/en-us/office/insert-mail-merge-fields-9a1ab5e3-2d7a-420d-8d7e-7cc26f26acff or https://docs.telerik.com/devtools/document-processing/libraries/radwordsprocessing/concepts/fields/merge-field)
- date need to be passed into the DynamicAlert control (i.e. ` dateTimestamp={nextIntake?.openTimestamp}`
